src/components/ActionBar/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './scss/ActionBar.scss';
+
+class ActionBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isShowing: false,
+    };
+  }
+
+  componentDidMount = () => {
+    if (this.props.messages.length) {
+      this.show(this.props.visibleDuration);
+    }
+  };
+
+  componentDidUpdate = prevProps => {
+    if (this.props.messages.length !== prevProps.messages.length) {
+      this.show(this.props.visibleDuration);
+    }
+  };
+
+  componentWillUnmount = () => {
```

Comment:
I don't think we should leave in lifecycles that have been announced to be deprecated

Reply:
true, but doesn't apply to this function

Reply:
why does it not apply?